packages/tempus/src/Tempus_StepperForwardEuler_impl.hpp

```diff
@@ -48,8 +49,29 @@ StepperForwardEuler<Scalar>::StepperForwardEuler(
     this->initialize();
   }
 }
+#endif
 
+template<class Scalar>
+StepperForwardEuler<Scalar>::StepperForwardEuler(
+  const Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> >& appModel,
+  bool useFSAL,
+  std::string ICConsistency,
+  bool ICConsistencyCheck,
+  const Teuchos::RCP<StepperForwardEulerAppAction<Scalar> >& stepperFEAppAction)
+{
+  this->setStepperType(        "Forward Euler");
+  this->setUseFSAL(            useFSAL);
+  this->setICConsistency(      ICConsistency);
+  this->setICConsistencyCheck( ICConsistencyCheck);
```

Comment:
I found I needed to also construct an observer here also to pass some tests (needed the deprecated ifdef), i.e., failed the initialization tests.